content/blog/2017/08/2017-08-05-remoting-update.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Remoting Update. Protocols deprecation, Java 8 requirement and plans"
+tags:
+- remoting
+- agents
+author: oleg_nenashev
+---
+
+[NOTE]
+====
+This is a post by link:https://github.com/oleg-nenashev[Oleg Nenashev],
+who is an active Jenkins core contributor.
+He is also responsible for Jenkins Remoting maintenance at link:http://cloudbees.com[CloudBees, Inc].
+You can follow him on Twitter: link:http://twitter.com/oleg_nenashev[@oleg_nenashev].
+====
+
+You may have noticed that there is some ongoing work on 
+link:https://github.com/jenkinsci/remoting/blob/master/README.md[Jenkins Remoting] in the project.
+On the other hand you may wonder where is the documentation and roadmaps.
+And I have to admit I was not that good on this front, but I want to make the activities more public.
+
+In order to achieve it, I have started the 
+link:https://jenkins.io//projects/remoting/[Remoting sub-project] on Jenkins website. 
+I will also do my best to make regular updates about the Remoting status in blogposts and the link:/mailing-lists[developer mailing list].
+Below you can find updates about incoming changes.
+
+## Remoting Protocols Deprecation
+
+It is almost one year since the release of JNLP4-connect protocol in Remoting 3.0. 
+This protocol has been enabled by default since 2.46.x, and so far it demonstrates good stability being compared to JNLP2 and especially to JNLP3.
+
+At the governance meeting 
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2017/jenkins-meeting.2017-08-02-18.00.html[we decided] to disable old Remoting protocols (JNLP/JNLP2 + CLI1) in new installations by default. 
+There are 3 reasons for it:
```

Comment:
When stating "Here are N reasons" it's better to use an ordered list, rather than unordered, bullet points.